### PR TITLE
COL-1155, script move_canvas_files_to_amazon_s3 will migrate whiteboard image_url files

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/COL-1155

* two SQL queries with narrow result-sets
* generic `moveFileToAmazonS3` function
* asset and whiteboard files listed in same CSV

### CODE REVIEW

**Comment by johncrossman** [comment]
Pardon the amendments just thrown in. Certain copy/pasted log statements needed wording change.

File: scripts/move_canvas_files_to_amazon_s3.js [hunk 1]
```diff
@@ -186,42 +186,94 @@ var storeAssetInAmazonS3 = function(asset, stream, filePath, callback) {
 };
 
 /**
- * Move file asset to Amazon S3.
+ * @param  {Course}       whiteboard       Whiteboard with image URL
+ * @param  {Course}       stream           The content being moved (input stream)
+ * @param  {Course}       filePath         Where to write content on local disk
```
Comment (paulkerschen) [inline scripts/move_canvas_files_to_amazon_s3.js]: @johncrossman Minor - `{Course}` not courses

Comment (johncrossman) [reply scripts/move_canvas_files_to_amazon_s3.js]: Thanks, @paulkerschen  Fixed.

**Comment by paulkerschen** [comment]
Thanks, @johncrossman, looks good to me. Will merge post Travis.

**Pull request merged**